.github/scripts/install.sh

```diff
@@ -38,6 +38,9 @@ if [ "$USE_CONDA" = "true" ]; then
     else
         mamba install 'numpy<1.23'
     fi
+
+    # Install docstring-to-markdown until we release PyLSP 1.6.0
+    mamba install docstring-to-markdown
```

Comment:
@dalthviz, this needs to be removed when updating our dependencies before releasing 5.4.0.